uwblueprint/focus-on-nature
# Feat: navbar functionality and visuals

## Notion ticket link

[Navbar: Tab navigation + visuals](https://www.notion.so/uwblueprintexecs/Navbar-Tab-navigation-visuals-9180da9b51a14b5ea4aa06b1031812ab)
Check the additional notes section for documented changes and updates to the original scope of this ticket



## Implementation description
* Created a navbar component in `NavBar.tsx` and additional pages for Camps, Global Forms and Access Control
* Routed the different navbar tabs to redirect to the specific pages
* Logout functionality included in the dropdown button with the user's name
* Styled according to Figma designs
* Added additional libraries (e.g. font awesome icons)



## Steps to test
1. **There's a logout bug if you're logged in for too long (known issue), so make sure you're logged out of the app before you test**
2. Go to localhost:3000 in your browser and login via Google
3. Check that the NavBar styling matches the Figma designs and that each tab redirects to its respective page (both page content and in the URL). The tab should be highlighted in green when you're on the specific page. Note: the logout button doesn't match the designs exactly, but the designers said it's ok as is.
5. Close the browser window. Go to localhost:3000 and check that you're redirected to the "/camps" page.
6. Open the dropdown menu on the right side and click to logout (check that you're redirected to the login page).
7. Make sure to test both admin and camp coordinator views



## What should reviewers focus on?
* The dependencies in the JSON files...not sure if some of them should be there
* Anything else we could do to improve on code quality


## Checklist
